relayer/src/target.ts

```diff
@@ -62,20 +64,31 @@ class Target {
         // if nonce is -1 API will do the lookup for the right value
         // https://polkadot.js.org/docs/api/cookbook/tx/#how-do-i-take-the-pending-tx-pool-into-account-in-my-nonce
         .signAndSend(this.signer, { nonce: -1 }, Promise.resolve)
-        .pipe(take(2)) // we only need to subscribe until second status - IN BLOCK
+        .pipe(
+           // we only need to subscribe until second status - IN BLOCK
+          take(2),
```

Comment:
Maybe check actual status, otherwise if new status is introduced then it will break in a way that will be hard to trace.